app/views/filters/_form.html.erb

```diff
@@ -48,7 +48,8 @@
 
         <%= autocomplete_f f, :search,
                            :disabled => f.object.unlimited?,
-                           :path => resource_path(f.object.resource_type) || "",
+                           :path => "",
```

Comment:
Ah, I hadn't realised we had another helper inbetween.  It may be better to check that options doesn't contain full_path here (https://github.com/theforeman/foreman/pull/2394/files#diff-d07b3cdd6c00768e06bfed349d3c808fR146) so as not to pass in an empty string.